django/contrib/sessions/backends/base.py

```diff
@@ -147,24 +156,60 @@ def is_empty(self):
         except AttributeError:
             return True
 
+    def _get_session_key_part(self, session_key):
+        "Return a string of the session_key key."
+        try:
+            algorithm, key = session_key.split(SESSION_KEY_DELIMITER)
+            if algorithm == SESSION_HASHING_ALGORITHM:
+                return key
+            return None
+        except ValueError:
+            return None
```

Comment:
This can be simplified to:

```python
if session_key.startswith(SESSION_HASHED_KEY_PREFIX):
    return session_key[len(SESSION_HASHED_KEY_PREFIX):]
```

That version will also be faster.